data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+			choice
+				`	"Outcome?"`
+				`	(Leave.)`
+					decline
+			`	"Better guns! These new ones will be perfect against those who dared to occupy our old worlds."`
+			choice
+				`	(Celebrate.)`
+			label celebrate
+			`	Now that the celebration is advancing, you realize you are the only one without a drink. Which beverage do you take? The green one is the one Sha-Ri was taking when you first came here, now he has a purple one.`
+			choice
+				`	(The purple one.)`
+				`	(The green one.)`
+					goto green
+				`	(Neither.)`
+					goto none
+			`	The purple beverage seems to make you dizzy, but you manage it well enough.`
+				goto continue
+			label green
+			`	The green beverage is very exciting, and keeps you invigorated for hours.`
+			label continue
+			`	You continue the party late into the night. There won't be any way to get information out of Sa-Ri now - he's heavily inebriated. After several hours, you leave with a slight headache. These Unfettered drinks are stronger than you thought, but you don't feel drunk in the way human drinks would make you.`
+				decline
+			label none
+			`	Even without a drink you can celebrate! You continue the party until late, even though some Unfettered do not seem to appreciate that you drink very little, if at all. You leave after several hours and get a good night's rest.`
+
+
+
+mission "Unfettered: The Word Has Spread"
+	invisible
+	source "Firelode"
+
+	to offer
+		has "Unfettered: Solifuge 5: Materials For Ships: done"
+		has "passenger space" >= 66
+
+	on offer
+		conversation
+			`Even before exiting your ship, you notice that the recent activity has not diminished. People are still moving around everywhere, getting to their ships or simply doing their jobs with renewed purpose. In all that activity you find your Chief friend, Sha-Ri, who seems happy to see you. "Ah, here you are when you are most needed, like always, <first>! Many True Hai have felt the calling of battle now, or the need to change. So many, in fact, that the sympathisers to our cause cannot keep up. Indeed those who join generally do so without a ship or many funds; what they are looking for is a chance at a new life, but they need help getting here. Could you provide that help?"`
+			choice
+				`	"Sure!"`
+					goto details
+				`	"Why do they want to join you?"`
+				`	"What battle are you talking about?"`
+					goto battle
+				`	"No way. I'm not helping in your crusade."`
+					decline
+			label why
+			`	"It depends. Some want to utilize the resources of the new worlds that are ready for the taking. Others seek inspiration by seeing our warriors fight. Others still want to join us as civilians, now that they're convinced the living conditions will not be as bad in our territory, or as health specialists to mitigate the loss of life. They want change, and we offer that change. They may have all they could possibly need at home, but that doesn't mean they feel like they have a purpose - or that they are fulfilling it."`
```

Comment:
There hasn't been enough mention of health or living conditions improving before this point for me to really believe this is the case. Maybe there should be missions where the player helps improve conditions by bringing materials specifically for improving infrastructure and such.

Reply:
You do bring medical supplies already
There are jobs for bringing materials but not an actual mission and the jobs were mostly though for shipyard materials, it would be nice to see them having to use those materials for habitations and such, earlier on, indeed

Reply:
I changed `Unfettered: Escort Hai Fleet` to say that you're bringing medicals teams and equipment to rebuild stuff, that's a start already